zerver/views/webhooks/trello/card_actions.py

```diff
@@ -0,0 +1,212 @@
+from six import text_type
+from typing import Dict, Tuple, Any, Optional, MutableMapping, Mapping
+from datetime import datetime
+from .exceptions import UnknownUpdateCardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_CARD_ACTIONS = [
+    u'updateCard',
+    u'createCard',
+    u'addLabelToCard',
+    u'removeLabelFromCard',
+    u'addMemberToCard',
+    u'removeMemberFromCard',
+    u'addAttachmentToCard',
+    u'addChecklistToCard',
+    u'commentCard'
+]
+
+CREATE = u'createCard'
+CHANGE_LIST = u'changeList'
+CHANGE_NAME = u'changeName'
+ARCHIVE = u'archiveCard'
+REOPEN = u'reopenCard'
+SET_DUE_DATE = u'setDueDate'
+CHANGE_DUE_DATE = u'changeDueDate'
+REMOVE_DUE_DATE = u'removeDueDate'
+ADD_LABEL = u'addLabelToCard'
+REMOVE_LABEL = u'removeLabelFromCard'
+ADD_MEMBER = u'addMemberToCard'
+REMOVE_MEMBER = u'removeMemberFromCard'
+ADD_ATTACHMENT = u'addAttachmentToCard'
+ADD_CHECKLIST = u'addChecklistToCard'
+COMMENT = u'commentCard'
+
+TRELLO_CARD_URL_TEMPLATE = u'[{card_name}]({card_url})'
+
+ACTIONS_TO_MESSAGE_MAPPER = {
+    CREATE: u'created {card_url_template}',
+    CHANGE_LIST: u'moved {card_url_template} from {old_list} to {new_list}',
+    CHANGE_NAME: u'renamed the card from "{old_name}" to {card_url_template}',
+    ARCHIVE: u'archived {card_url_template}',
+    REOPEN: u'reopened {card_url_template}',
+    SET_DUE_DATE: u'set due date for {card_url_template} to {due_date}',
+    CHANGE_DUE_DATE: u'changed due date for {card_url_template} from {old_due_date} to {due_date}',
+    REMOVE_DUE_DATE: u'removed due date from {card_url_template}',
+    ADD_LABEL: u'added a {color} label with \"{text}\" to {card_url_template}',
+    REMOVE_LABEL: u'removed a {color} label with \"{text}\" from {card_url_template}',
+    ADD_MEMBER: u'added {member_name} to {card_url_template}',
+    REMOVE_MEMBER: u'removed {member_name} from {card_url_template}',
+    ADD_ATTACHMENT: u'added [{attachment_name}]({attachment_url}) to {card_url_template}',
+    ADD_CHECKLIST: u'added {checklist_name} checklist to {card_url_template}',
```

Comment:
@TomaszKolek Now that I see how this is used, I believe the phrasing should be:

'added the {checklist_name} checklist to {card_url_template}'